app/app.js

```diff
@@ -3,14 +3,14 @@ import Resolver from 'ember/resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
 
-var App;
+let App;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).